tests/impact/client/test_client.py

```diff
@@ -10,7 +10,9 @@ def test_create_workspace(create_workspace):
         url=create_workspace.url, context=create_workspace.context
```

Comment:
This file need the import:
`import modelon.impact.client.entities`
to fix some lint errors